xml/security_preface.xml

```diff
@@ -99,7 +99,8 @@
   <title>Passwords</title>
   <para>
    On a Linux system, only hashes of passwords are stored. Hashes are
-   one-way algorithms that make it easy to encrypt data. At the same
+   one-way algorithms which scramble data to a digital fingerprint that
+   is hard to reverse. At the same
    time, hash algorithms make it very hard to compute the original
    secret from the hash.
```

Comment:
@denknorr : After rephrasing the sentence before, can we scratch the next sentence?

Reply:
Thx, done now - I will merge and backport now